test/e2e/framework/apps/doc.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2020 TriggerMesh Inc.
```

Comment:
Some years are kept 2020, and I think that's ok since most of this contents are being just moved.

Reply:
Right, I didn't bother updating years. Should I?

Reply:
IMHO 2020 is perfectly ok